rules/linux/defense_evasion_rename_esxi_files.toml

```
@@ -49,13 +52,33 @@ For more details on Elastic Agent configuration settings, refer to the [helper g
 For more details on Elastic Defend refer to the [helper guide](https://www.elastic.co/guide/en/security/current/install-endpoint.html).
 """
 severity = "medium"
-tags = ["Domain: Endpoint", "OS: Linux", "Use Case: Threat Detection", "Tactic: Defense Evasion", "Data Source: Elastic Defend"]
+tags = [
+    "Domain: Endpoint",
+    "OS: Linux",
+    "Use Case: Threat Detection",
+    "Tactic: Defense Evasion",
+    "Data Source: Elastic Defend",
+]
 timestamp_override = "event.ingested"
 type = "eql"
+
 query = '''
 file where host.os.type == "linux" and event.action == "rename" and
-file.Ext.original.name : ("*.vmdk", "*.vmx", "*.vmxf", "*.vmsd", "*.vmsn", "*.vswp", "*.vmss", "*.nvram", "*.vmem")
-and not file.name : ("*.vmdk", "*.vmx", "*.vmxf", "*.vmsd", "*.vmsn", "*.vswp", "*.vmss", "*.nvram", "*.vmem")
+
+    /* filter for ESXI file extensions in the original file name */
+    (endsWith(file.Ext.original.name, ".vmdk") or
+    endsWith(file.Ext.original.name, ".vmx") or
+    endsWith(file.Ext.original.name, ".vmxf") or
+    endsWith(file.Ext.original.name, ".vmsd") or
+    endsWith(file.Ext.original.name, ".vmsn") or
+    endsWith(file.Ext.original.name, ".vswp") or
+    endsWith(file.Ext.original.name, ".vmss") or
+    endsWith(file.Ext.original.name, ".nvram") or
+    endsWith(file.Ext.original.name, ".vmem")
+    ) and
```

```suggestion
file.Ext.original.name : ("*.vmdk", "*.vmx", "*.vmxf", "*.vmsd", "*.vmsn", "*.vswp", "*.vmss", "*.nvram", "*.vmem") and
```
IMO, I think we should maintain the original syntax as this makes it more complex